docs/reference/configuration/eebus.md

```diff
@@ -6,6 +6,9 @@ sidebar_position: 13
 
 :::warning
 EEBUS Unterstützung ist noch im experimentellen Stadium. Es wird hier sicher zu dem einen oder anderen Problem kommen!
+
+Wenn EEBUS über Home Assistant genutzt wird, muss unbedingt das Interface "end0" angegeben und die IP-Adresse von Home Assistant im URI (192.168.1.216:4712) eingetragen werden.
+Sehe [Issue](https://github.com/evcc-io/hassio-addon/issues/99)
```

Comment:
Bei dem Satz wird mich nicht ganz klar, welches Szenario hier adressiert wird. Gehts um ein evcc, das in Home Assistant läuft? Wo genau muss Interface und IP-Adresse eingegeben werden?

Reply:
Ich habe ein Vodafone Router, er macht ipv6 + ipv4 was dann über das protokol mdns exposed wird. evcc kommt damit nicht zu recht. Sehe Issue 99. 

Setup: Evcc auf hassio als container